services/wikiClient.test.js

```diff
@@ -0,0 +1,66 @@
+const axios = require('axios');
+
+const WikiClient = require('./wikiClient');
+
+jest.mock('axios');
+
+describe('WikiClient', () => {
+  it('initializes a new axios client with defaults for header and timeout', () => {
+    const headers = {
+      common: {
+        'Content-Type': 'application/json; charset=utf-8',
+        Accept: 'application/json',
+      },
+    };
+    const timeout = 5000;
+
+    const subject = new WikiClient(); // eslint-disable-line no-unused-vars
```

Comment:
You can make this line linter-compliant by simply writing `new WikiClient()` without the assignment.
Should also work for the test!

Reply:
Unfortunately not 😿 

I already tried this and it then violates [another cop](https://eslint.org/docs/rules/no-new)... Fully ditching `new` breaks the test though...

Reply:
Bummer! I guess then it is alright as it is :)

Reply:
You might want to add an expectation that the `subject.client` is set to whatever is returned from `axios.create`. That solves both your linter issue and arguably makes it clearer what the `axios.create` call is intended for.

À la 👨‍🍳 

```javascript
const instance = { dummy: true };
axios.create.mockReturnValue(instance);

// …

expect(subject.client).toBe(instance);
```

Reply:
Purrfect @pmeinhardt 💚